pom.xml

```diff
@@ -70,44 +67,65 @@
     </dependencies>
 
     <build>
-        <finalName>${project.artifactId}</finalName>
-
-        <pluginManagement>
-            <plugins>
-                <plugin>
-                    <groupId>net.wasdev.wlp.maven.plugins</groupId>
-                    <artifactId>liberty-maven-plugin</artifactId>
-                    <version>1.1</version>
-                </plugin>
-            </plugins>
-        </pluginManagement>
         <plugins>
-        
-              <plugin>
-                    <groupId>org.apache.maven.plugins</groupId>
-                    <artifactId>maven-war-plugin</artifactId>
-                    <version>2.3</version>
-                    <configuration>
-                        <!-- Ignore error about missing web.xml -->
-                        <failOnMissingWebXml>false</failOnMissingWebXml>
-                        <archive>
-                            <!-- Do not include META-INF/maven to avoid long file problems on windows -->
-                            <addMavenDescriptor>false</addMavenDescriptor>
-                        </archive>
-                    </configuration>
-                </plugin>
-        
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-war-plugin</artifactId>
+                <version>2.3</version>
+                <configuration>
+                    <!-- Ignore error about missing web.xml -->
+                    <failOnMissingWebXml>false</failOnMissingWebXml>
+                    <archive>
+                        <!-- Do not include META-INF/maven to avoid long file problems on windows -->
+                        <addMavenDescriptor>false</addMavenDescriptor>
+                    </archive>
+                </configuration>
+            </plugin>
             <plugin>
                 <groupId>net.wasdev.wlp.maven.plugins</groupId>
                 <artifactId>liberty-maven-plugin</artifactId>
+                <version>2.0-SNAPSHOT</version>
                 <configuration>
                     <configFile>src/main/wlp/server.xml</configFile>
+                    <installAppPackages>project</installAppPackages>
+                    <looseApplication>true</looseApplication>
+                    <type>zip</type>
```

Comment:
You can delete line 90 `<type>zip</zip>` which is only used within the `<install>` or `<assemblyArtifact>` element.

see https://github.com/WASdev/ci.maven/blob/master/docs/installation-configuration.md